app/views/mobile_dashboard/_income.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="income-info">
+  <div class="half">
+    <h4><span>Country</span> Income</h4>
+    Credits: #<br>Income Level: #</div>
+  <div class="half">
+    <label for="incomeTurn">
+      <span class="meta">Income for Turn</span>
+      <select id="incomeTurn" ng-change="updateFilter()" ng-model="selectedRound">
+        <option ng-repeat="round in allRounds" value="{{round}}" selected="selectedRound('{{round}}')">{{round}}</option>
```

Comment:
Remaining bug: Because <option> is generated by rounds, there is a thing that occurs where it creates a `null` option before rounds are calculated. Working on this one.